app/services/alerts/generate_email_notifications.rb

```diff
@@ -4,9 +4,12 @@ def perform
       Contact.all.each do |contact|
         events = contact.alert_subscription_events.where(status: :pending, communication_type: :email)
         if events.any?
+          message_id = SecureRandom.uuid
           alerts = events.map(&:alert)
-          AlertMailer.with(email_address: contact.email_address, alerts: alerts, school: contact.school).alert_email.deliver_now
-          events.update_all(status: :sent)
+          alert_ids_as_string_parameter = alerts.pluck(:id).to_s
```

Comment:
Not sure I follow why these need to be a string?

Reply:
Yep, will sort this as discussed